src/git.ts

```diff
@@ -108,9 +108,25 @@ export async function getNewCommits(since?: Commit): Promise<CommitSequence> {
     if (since) {
         args.push(headBehindLastMigration ? 'HEAD..' + since.sha1 : since.sha1 + '..HEAD');
     }
-    const [stdout] = await execFile('git', args);
-    const output = stdout.toString().trim();
-    const commits = parseGitLog(output);
+    const commits = await (new Promise(async (resolve, reject) => {
+        const gitProcess = await spawn('git', args);
```

Comment:
if this rejects, the error will not be handled

Reply:
is `spawn` even async?

Reply:
True, I'll add error handling

see `spawnSync`

Reply:
It's not https://github.com/normalize/mz/blob/master/child_process.js